llvm/lib/Target/AArch64/AArch64InstrFormats.td

```diff
@@ -5985,6 +5985,26 @@ multiclass SIMDThreeSameVectorFP<bit U, bit S, bits<3> opc,
         [(set (v2f64 V128:$Rd), (OpNode (v2f64 V128:$Rn), (v2f64 V128:$Rm)))]>;
 }
 
+let mayRaiseFPException = 1, Uses = [FPCR] in
+multiclass SIMDThreeVectorFP<bit U, bit S, bits<3> opc,
```

Comment:
Why are you adding this class, it is not used by 
defm FAMAX : SIMDThreeSameVectorFP

Reply:
Removed.